todo_app/.env.test.py

```diff
@@ -0,0 +1,15 @@
+# Flask server configuration.
+FLASK_SKIP_DOTENV=True
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+TOKEN=vg228f78a50a5c6745b733c282fkcd3953c04b118ctf893a12b439340a5ca3 
+KEY=52r5ab920p78a542ed2fb5541190d22d     
```

Comment:
(Same comment on trying to avoid fake credentials that look real!)

Reply:
Updated,thank you